26_Autoexposure/app_resources/luma_meter.comp.hlsl

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2024-2024 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#include "nbl/builtin/hlsl/luma_meter/luma_meter.hlsl"
+#include "nbl/builtin/hlsl/bda/bda_accessor.hlsl"
+#include "app_resources/common.hlsl"
+
+[[vk::combinedImageSampler]] [[vk::binding(0, 0)]] Texture2D texture;
+[[vk::combinedImageSampler]] [[vk::binding(0, 0)]] SamplerState samplerState;
+
+[[vk::push_constant]] AutoexposurePushData pushData;
+
+using Ptr = nbl::hlsl::bda::__ptr < uint32_t >;
```

Comment:
pro tip, you can actually put `using namespace nbl::hlsl; in your "main" hlsl files (the ones with entry points)